src/libs/fileDownload/validateReceiptFile/index.native.ts

```diff
@@ -0,0 +1,24 @@
+import checkFileExists from '@libs/fileDownload/checkFileExists';
+
+/**
+ * Validates a receipt file and processes it for upload (Native implementation)
+ * Uses checkFileExists for memory-efficient file validation without loading the entire file
+ */
+function validateReceiptFile(
+    itemReceiptFilename: string | undefined,
+    itemReceiptPath: string | undefined,
+    itemReceiptType: string | undefined,
```

Comment:
let's rename these to just `receiptXYZ` as `item` doesn't add much value here


```suggestion
    receiptFilename: string | undefined,
    receiptPath: string | undefined,
    receiptType: string | undefined,
```

Reply:
Same for the other instances of `itemReceiptXYZ` in other files